MANIFEST.in

```diff
@@ -32,6 +32,7 @@ recursive-include invenio_access *.po
 recursive-include invenio_access *.pot
 recursive-include invenio_access *.py
 recursive-include invenio_access *.mo
+include *.db
```

Comment:
You don't want to include any `*.db` in the package. It means that your tests didn't clean  well after running.